instrumentation/apache-dubbo-2.7/javaagent/src/test/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboAgentTest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.apachedubbo.v2_7;
+
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class DubboAgentTest extends AbstractDubboTest {
```

Comment:
i think these test classes can be package private instead of public

Reply:
Yes, i  will remove `public ` for unit test class and method